spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -312,10 +312,10 @@ def step(  # noqa
                 if self.config.TERMINATE_ON_GRASP:
                     self.should_end = True
         elif place:
-            print("PLACE ACTION CALLED: Opening the gripper!")
+            self.say("PLACE ACTION CALLED: Opening the gripper!")
             if self.get_grasp_angle_to_xy() < np.deg2rad(30):
                 self.turn_wrist()
-                print("open gripper in place")
+                self.say("open gripper in place")
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.